### PR TITLE
[sc-10905] REMi metrics UI

### PR DESCRIPTION
**Pull request by mpellerin42**

### CODE REVIEW

File: libs/common/src/assets/i18n/fr.json [hunk 1]
```diff
@@ -569,6 +571,21 @@
 	"kb.form.invalid-slug": "Knowledge Box slug invalide",
 	"kb.form.name": "Nom de la Knowledge Box",
 	"kb.form.required-name": "Champs requis",
+	"kb.metrics.health-status.description": "Voyez si votre pipeline RAG est en bonne santé en vérifiant les valeurs moyennes, minimales et maximales de chaque indicateur sur la période sélectionnée.",
```
Comment (ebrehault) [inline libs/common/src/assets/i18n/fr.json:574]:
```suggestion
	"kb.metrics.health-status.description": "Voir si votre pipeline RAG est en bonne santé en vérifiant les valeurs moyennes, minimales et maximales de chaque indicateur sur la période sélectionnée.",
```
The infinitive sounds a bit better here, no?

File: libs/common/src/assets/i18n/fr.json [hunk 2]
```diff
@@ -569,6 +571,21 @@
...
+	"kb.metrics.missing-knowledge.criteria.context-lower-than-label": "Score de pertinence du contexte inférieur à",
+	"kb.metrics.missing-knowledge.criteria.month.format": "Format : AAAA-MM",
+	"kb.metrics.missing-knowledge.criteria.month.label": "Pour le mois",
+	"kb.metrics.missing-knowledge.description": "Découvrez les connaissances manquantes dans votre Knowledge Box en examinant les questions pour lesquelles le score de pertinence du contexte est mauvais.",
```
Comment (ebrehault) [inline libs/common/src/assets/i18n/fr.json:578]:
```suggestion
	"kb.metrics.missing-knowledge.description": "Identifier les connaissances manquantes dans votre Knowledge Box en examinant les questions pour lesquelles le score de pertinence du contexte est mauvais.",
```

File: libs/common/src/assets/i18n/fr.json [hunk 3]
```diff
@@ -569,6 +571,21 @@
...
+	"kb.metrics.missing-knowledge.title": "Connaissances manquantes dans ma Knowledge Box",
+	"kb.metrics.page-description": "Évaluez les performances de votre pipeline RAG avec <a href=\"https://nuclia.com/developers/remi-open-source-rag-evaluation-model/\" target=\"_blank\">REMi</a>.",
```
Comment (ebrehault) [inline libs/common/src/assets/i18n/fr.json:580]:
```suggestion
	"kb.metrics.page-description": "Évaluer les performances de votre pipeline RAG avec <a href=\"https://nuclia.com/developers/remi-open-source-rag-evaluation-model/\" target=\"_blank\">REMi</a>.",
```

File: libs/common/src/assets/i18n/fr.json [hunk 4]
```diff
@@ -569,6 +571,21 @@
...
+	"kb.metrics.page-title": "Mesures d'évaluation RAG (REMi)",
+	"kb.metrics.performance-evolution.description": "Suivez l'évolution des performances de votre pipeline RAG",
```
Comment (ebrehault) [inline libs/common/src/assets/i18n/fr.json:582]:
```suggestion
	"kb.metrics.performance-evolution.description": "Suivre l'évolution des performances de votre pipeline RAG",
```

File: libs/common/src/lib/metrics/metrics-page.component.ts [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+  PaTextFieldModule,
+} from '@guillotinaweb/pastanaga-angular';
+import { Observable, Subject } from 'rxjs';
+import { DatedRangeChartData, RangeChartComponent, RangeChartData, RangeEvolutionChartComponent } from '../charts';
+import { RemiMetricsService, RemiPeriods } from './remi-metrics.service';
+import {
+  RemiQueryCriteria,
+  RemiQueryResponseContextDetails,
+  RemiQueryResponseItem,
+} from '../../../../sdk-core/src/lib/db/kb/activity/remi.models';
```
Comment (ebrehault) [inline libs/common/src/lib/metrics/metrics-page.component.ts:26]: wrong import path

**Review by ebrehault — APPROVED** [review]

File: apps/dashboard/src/app/knowledge-box/knowledge-box-home/knowledge-box-home.component.html [hunk 6]
```diff
@@ -269,104 +269,124 @@
...
+        @if (isKbContrib | async) {
+          @if (isRemiMetricsEnabled | async) {
+            <nsi-home-container>
+              <div class="title-with-action">
+                <div class="title-with-icon title-m">
+                  <pa-icon name="heart"></pa-icon>
+                  {{ 'dashboard-home.health-status.title' | translate }}
+                  <small class="body-s">({{ 'kb.metrics.period.last-7d' | translate }})</small>
+                </div>
+                <a
```
Comment (operramon) [inline apps/dashboard/src/app/knowledge-box/knowledge-box-home/knowledge-box-home.component.html:281]: A minor thing, it seems that this link is visible by "writer" users , but hey are not allowed to access the metrics page.

**Review by operramon — APPROVED** [review]

File: apps/dashboard/src/app/knowledge-box/knowledge-box-home/knowledge-box-home.component.html (hunk 6 above)
Comment (mpellerin42) [reply apps/dashboard/src/app/knowledge-box/knowledge-box-home/knowledge-box-home.component.html:281]: oh, good catch! thanks

Comment (mpellerin42) [reply apps/dashboard/src/app/knowledge-box/knowledge-box-home/knowledge-box-home.component.html:281]: fixed

**Pull request merged**